server/models/UserRequests.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const userRequestsSchema = new mongoose.Schema({
+  user_id: {
+    type: Number,
+    required: true,
+    unique: true
+  },
+  sitter_id: {
+    type: Number,
+    required: true,
+    unique: true
+  },
+  start_date: {
+    type: Date,
+    default: Date.now(),
+  },
+  end_date: {
+    type: Date,
+    //7 Days later
+    default: () => Date.now() + 7*24*60*60*1000,
+  },
+  accepted: {
+    type: Boolean,
+    default: false,
+  },
+  declined: {
+    type: Boolean,
+    default: false,
+  },
```

Comment:
I may have mentioned this before, I'd suggest a single `status` field with an `enum` validator could be more appropriate for storing this information - though I'll leave it up to you.